library/scrollphathd/__init__.py

```diff
@@ -1,5 +1,6 @@
 import atexit
 from sys import exit, version_info
+from .api.api import start_background_thread, scrollphathd_blueprint
```

Comment:
Why `.api.api`? I'd favour stuffing everything into `__init__` so you can simply `from api import x` or `import scrollphathd.api` or even `python -m scrollphathd.api`, although I may be violating some Python tenant by doing so?

Actually having said this, would it make sense to instead rename `api.py` to `http.py`, allowing for the possible future inclusion of other API types such as socket, etc?

Reply:
Can do either, tho I would avoid having scrollphathd.api be used in the “python -m” way, tho I guess it’s not so bad if you use if main blablabla. Have you used entrypoints before? That’s the more pythonic way of doing it I think, I’ll stick one in this PR in the morning

Mostly I just avoid sticking stuff in __init__, people tend to do dangerous stuff like run code on import. Not sticking it in init avoids the temptation. However it’s your library so I’m happy with either init or http.

Reply:
>  people tend to do dangerous stuff like run code on import

Lalalalala...lalala....ala....

/me hides

Reply:
Perhaps `http` is the right choice, it's descriptive and allows for branching out in future with sockets, mtp, etc if we feel crazy enough to build that stuff in!

I've had a bit of a run-in with the ol' "run code on import" trap and broke things. See: https://www.raspberrypi.org/forums/viewtopic.php?f=32&t=193502&p=1212488#p1212488